src/modules/status-list/status-list.controller.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  Body,
+  Controller,
+  Get,
+  HttpStatus,
+  Param,
+  Post,
+  Res,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { ApiResponse, ApiTags } from '@nestjs/swagger';
+import { Response } from 'express';
+import { User } from '../../common/user.decorator';
+import { RoleService } from '../role/role.service';
+import { CreateEntryInputDto } from './dtos/create-entry-input.dto';
+import { CredentialWithStatusDto } from './dtos/credential-status.dto';
+import { RegisterRevokeInputDto } from './dtos/register-revoke-input.dto';
+import { SignRevokeInputDto } from './dtos/sign-revoke-input.dto';
+import { StatusListCredentialDto } from './dtos/status-list-credential.dto';
+import { StatusListVerifiableCredentialDto } from './dtos/status-list-verifiable-credential.dto';
+import { STATUS_LIST_MODULE_PATH } from './status-list.const';
+import { StatusListService } from './status-list.service';
+
+@Controller({ path: STATUS_LIST_MODULE_PATH, version: '1' })
+@ApiTags('status-list')
+export class StatusListController {
+  constructor(
+    private readonly statusListService: StatusListService,
+    private readonly roleService: RoleService
+  ) {}
+
+  @Post('/create-entry')
```

Comment:
As it is a `POST` already, I think that we don't need to specify `create`.
```suggestion
  @Post('/entries')
```